include/zephyr/net/conn_mgr_connectivity.h

```diff
@@ -34,7 +34,7 @@ extern "C" {
 #define _NET_MGMT_CONN_LAYER			NET_MGMT_LAYER(NET_MGMT_LAYER_L2)
```

Comment:
One note about the commit messages. It now has duplicate info which is pointless and states the obvious.
So the commit subject is good, answering the question "what", but the commit message is missing the answer the question "why" or "why this commit exists". I am ranting about these things because if one looks the commit log in the future and wonders what / why some patch exists, it is difficult if the message does not tell it clearly.

Reply:
Would this be a more satisfying commit message to you?

```
conn_mgr events cannot be used due to botched event definitions.

Fix these definitions so that events can be used.

Also patch hole in test coverage that allowed this in the first place.
```

Reply:
> ```
> conn_mgr events cannot be used due to botched event definitions.
> 
> ```

I've learned a new word today :)

However, https://developers.google.com/tech-writing/one/audience#simple_words

Reply:
> Would this be a more satisfying commit message to you?

Yes, much better, thx :+1: